servlet/5.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Servlet 5.0 (under development)"
-date: 2020-01-15
+title: "Jakarta Servlet 5.0 (milestone)"
+date: 2020-06-12
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
 
 * [Jakarta Servlet 5.0 Release Record]()
```

Comment:
Same nit as the others... Include the URL for the Servlet release record from the PMI page.  Thanks!